shared/variables.tf

```diff
@@ -0,0 +1,25 @@
+variable "region" {
+  default     = "ap-southeast-1"
+  description = "AWS region"
+  type        = string
+}
+
+variable "environment" {
+  description = "The application environment, used to tag the resources, e.g. `acme-web-staging`"
+  type        = string
+}
+
+variable "owner" {
+  description = "The owner of the infrastructure, used to tag the resources, e.g. `acme-web`"
+  type        = string
+}
+
+variable "aws_access_key" {
```

Comment:
Just curious whether we actually need this inside the shared modules? How is the access key and secret intended to be used 🤔